### PR TITLE
updated_title_property_apidocs

### LINKED ISSUE #57673: [web] Page title is set via title property of MaterialApp
I use Firebase hosting to host my Flutter web app. When I run it on Chrome on Windows, the text on the Chrome tab is  "Flutter Demo". I don't know where that name comes from, and I want to see the name of my app instead.
Skärmklipp

My index.html:
```
<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
  <meta name="description" content="A new Flutter project.">

  <!-- iOS meta tags & icons -->
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  <meta name="apple-mobile-web-app-title" content="new_vem_bor_var">
  <link rel="apple-touch-icon" href="icons/Icon-192.png">

  <!-- Favicon -->
  <link rel="shortcut icon" type="image/png" href="favicon.png"/>

  <title>new_vem_bor_var</title>
  <link rel="manifest" href="manifest.json">
</head>
<body>
  <!-- This script installs service_worker.js to provide PWA functionality to
       application. For more information, see:
       https://developers.google.com/web/fundamentals/primers/service-workers -->
  <script>
    if ('serviceWorker' in navigator) {
      window.addEventListener('load', function () {
        navigator.serviceWorker.register('flutter_service_worker.js');
      });
    }
  </script>
  <script src="main.dart.js" type="application/javascript"></script>
</body>
</html>
```
My manifest.json:
```
{
    "name": "new_vem_bor_var",
    "short_name": "new_vem_bor_var",
    "start_url": ".",
    "display": "standalone",
    "background_color": "#0175C2",
    "theme_color": "#0175C2",
    "description": "A new Flutter project.",
    "orientation": "portrait-primary",
    "prefer_related_applications": false,
    "icons": [
        {
            "src": "icons/Icon-192.png",
            "sizes": "192x192",
            "type": "image/png"
        },
        {
            "src": "icons/Icon-512.png",
            "sizes": "512x512",
            "type": "image/png"
        }
    ]
}
```
Flutter version:
```D:\AndroidStudioProjects\new_vem_bor_var>flutter --version
Flutter 1.18.0-11.1.pre • channel beta • https://github.com/flutter/flutter.git
Framework • revision 2738a1148b (7 days ago) • 2020-05-13 15:24:36 -0700
Engine • revision ef9215ceb2
Tools • Dart 2.9.0 (build 2.9.0-8.2.beta)```

### PR DESCRIPTION
**Pull request by Aman9026**
## Description

updated title property apidoc for web for solving issue #57673

## Related Issues

Fixes #57673

## Checklist

Before you create this PR, confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/app.dart [hunk 1]
```diff
@@ -615,6 +615,7 @@ class WidgetsApp extends StatefulWidget {
   /// displayed when the user presses the "recent apps" button. On iOS this
   /// value cannot be used. `CFBundleDisplayName` from the app's `Info.plist` is
   /// referred to instead whenever present, `CFBundleName` otherwise.
+  /// On Web it appears as the Page title shown on the tab.
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/app.dart:618]: Maybe better: "On the web it is used as the page title, which shows up in the browser's list of open tabs."

**Review by goderbauer — APPROVED** [review]
LGTM

**Comment by Piinks** [comment]
Hey @Aman9026 can you update your branch with the latest from master? That should resolve the failing checks here. Thanks!

**Comment by Aman9026** [comment]
> Hey @Aman9026 can you update your branch with the latest from master? That should resolve the failing checks here. Thanks!

Done

**Comment by Oklahomawhore** [comment]
how to change title for individual pages?

**Pull request merged**